Assets/MirageXR/ContentTypes/VirtualInstructor/Editors/SpatialUI/AI/ContextPrompt.cs

```diff
@@ -23,6 +23,7 @@ public class ContextPromt : PopupBase
         public override void Initialization(Action<PopupBase> onClose, params object[] args)
         {
             base.Initialization(onClose, args);
+            TryToGetArguments(args);
```

Comment:
you shouldn't call `TryToGetArguments(args);`
it will be called from `base.Initialization(onClose, args);`
```
public virtual void Initialization(Action<PopupBase> onClose, params object[] args)
    {
        _onClose = onClose;
        if (!TryToGetArguments(args))
        {
            Debug.LogError("error when trying to get arguments!");
        }
    }
```